aether/sdk/tests/fakeapp/views.py

```diff
@@ -53,9 +55,10 @@ def custom_404(self, request, pk=None, *args, **kwargs):
         return Response(data=self.serializer_class(obj, context={'request': request}).data)
 
 
-class TestChildModelViewSet(MtViewSetMixin, ModelViewSet):
+class TestChildModelViewSet(MtViewSetMixin, FilteredMixin, ModelViewSet):
     queryset = TestChildModel.objects.order_by('name')
     serializer_class = TestChildModelSerializer
+    filter_class = TestChildModelFilter
```

Comment:
If you want to use this filter class you need to indicate that in the settings:
https://www.django-rest-framework.org/api-guide/filtering/#djangofilterbackend

To avoid install a new library (only for tests) why don't you use the `SearchFilter` class?
https://www.django-rest-framework.org/api-guide/filtering/#searchfilter

```python
search_fields = ['parent__name',]
```

In that case in the test the query string parameter is `search` :grimacing: